src/main/command/statuscheckcommands/StatusCheckCommand.ts

```diff
@@ -10,14 +11,18 @@ export class StatusCheckCommand extends Command {
                    userPerms: Permissions,
                    messageReply: Function,
                    ...args: CommandArgs[]): CommandResult {
-
+        //Calculation for converting Discord Uptime (milliseconds) into hours, minutes and seconds.
         const { uptime } = args[0] as StatusCheckCommandData;
+        //Hours
         let upTimeInHours = (uptime/1000) / 3600;
```

Comment:
Why don't you just do
` upTimeInHours = Math.floor(upTimeInHours)?`
Redeclaring a `rupTimeInHours` is unnecessary. This is the same for the hours and seconds.